projects/lf-documentation/src/app/getting-started-cdn-example/getting-started-cdn-example.component.html

```diff
@@ -0,0 +1,154 @@
+  >"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/indigo-pink.css"</span><span class="code-html-attribute-name"> rel</span><span
+  >=</span><span class="code-html-attribute-value">"stylesheet" </span><span class="code-html-tagname">/&gt;</span>
+    </pre>
+  </li>
+  <li>
+    If you are using TypeScript, add a file <code>./src/definitions.d.ts</code>. This enables custom elements in your React application.
+    <pre class="code-container">
+<span class="code-ts-import">declare namespace</span> JSX &#123;
+  <span class="code-ts-import">interface</span> IntrinsicElements &#123;
+    <span class="code-js-string">'lf-checklist'</span>: any;
+  }
+}
+    </pre>
+  </li>
+  <li>
+    <p dir="auto">In the <code>.js</code> file,</p>
+    <pre class="code-container">
+<span class="code-ts-import">import</span> <span class="code-js-import-item">React</span> <span class="code-ts-import">from</span> <span class="code-js-string">'react'</span>;
+<span class="code-ts-comment">// in tsx project, add the following import statements  </span>
+<span class="code-ts-import">import</span> <span class="code-js-import-item">&#123; NgElement, WithProperties }</span> <span class="code-ts-import">from</span> <span class="code-js-string">'@angular/elements'</span>;
+<span class="code-ts-import">import</span> <span class="code-js-import-item">&#123;LfChecklistComponent, Checklist }</span> <span class="code-ts-import">from</span> <span class="code-js-string">'@laserfiche/types-lf-ui-components'</span>;
+
+<span class="code-ts-import">export class</span> <span class="code-js-classname">App</span> <span class="code-ts-import">extends</span> <span class="code-js-classname">React</span>.<span class="code-js-prop">Component</span> &#123;
+  <span class="code-js-prop">checklist</span>;
+  <span class="code-ts-comment">// in tsx project, provide type </span>
+  <span class="code-ts-comment">// checklist: React.RefObject&lt;NgElement & WithProperties&lt;LfChecklistComponent&gt;&gt;;</span>
+
+  <span class="code-js-function">constructor</span>(props) &#123;
+    super(props);
+    <span class="code-js-prop">checklist</span> = <span class="code-js-classname">React</span>.<span class="code-js-function">createRef()</span>;  <span class="code-ts-comment">// create handle for custom element</span>
+  }
+
+  <span class="code-ts-import">async</span> <span class="code-js-function">componentDidMount()</span> &#123;
+    <span class="code-ts-import">await</span> this.<span class="code-js-prop">checklist.current</span>?.<span class="code-js-function">initAsync</span>(...);
+    this.<span class="code-js-prop">checklist.current</span>?.<span class="code-js-function">addEventListener</span>(<span class="code-js-string">'checklistChanged'</span>, this.<span class="code-js-prop">onChecklistChanged</span>);
+    <span class="code-ts-comment">// we have to add event listen to listen for the HTML custom event checklistChanged</span>
```

Comment:
... event listener...

This line should be above where you add the event listener